src/monoflow.ts

```diff
@@ -25,6 +25,10 @@ export class Workflow<Z, A, B> {
     return new Workflow(undefined, fn, [...this._steps, this]);
   }
 
+  combine<T>(workflow: Workflow<Z, B, T>): Workflow<Z, A, T> {
```

Comment:
I think it should be:

```ts
   combine<C>(workflow: Workflow<Z, B, C>): Workflow<Z, B, C> {
```

Z is the input type for the first step
B is the input for for the last step
C is the output type for the last step

Reply:
Thanks. I solved it.